src/backend/base/langflow/__main__.py

```diff
@@ -79,69 +79,73 @@ def set_var_for_macos_issue():
 
 @app.command()
 def run(
-    host: str = typer.Option("127.0.0.1", help="Host to bind the server to.", envvar="LANGFLOW_HOST"),
-    workers: int = typer.Option(1, help="Number of worker processes.", envvar="LANGFLOW_WORKERS"),
-    timeout: int = typer.Option(300, help="Worker timeout in seconds.", envvar="LANGFLOW_WORKER_TIMEOUT"),
-    port: int = typer.Option(7860, help="Port to listen on.", envvar="LANGFLOW_PORT"),
+    host: str | None = typer.Option(None, help="Host to bind the server to.", show_default=False),
```

Comment:
Not related to this PR but it would probably be nice to use the syntax `host: Annotated[str | None, typer.Option(None, help="Host to bind the server to.", show_default=False)]` = None` for all the params.

Reply:
I tried applying the suggestion, but the code started breaking. I'll take a closer look at it later.